tests/functional/raw-node/package.json

```diff
@@ -11,7 +11,7 @@
     "test-aws": "AWS_ON_AIR=true mocha -t 40000 test/",
     "test-gcp": "mocha -t 40000 test/GCP/",
     "test-routes": "mocha -t 40000 test/routes/",
-    "test": "mocha -t 40000 test/",
+    "test": "mocha -t 40000 test/routes/routeBackbeatForReplication.js",
```

Comment:
shouldn't this test be renamed as it's targetting now specific tests (unless this was just for testing purpose)

Reply:
Good catch, I force-pushed it unwillingly :) https://github.com/scality/cloudserver/compare/44a1c1c4b492ec1a7c16617a3ef91da7430ec290..6a37bf33d836379203a0049ebae8e411040f59fc